One-Foot-In/ofi-alumni
# Polling and Announcements

This PR allows admins to create a poll or an annoucement and send it out to users of OFI. The admin can target by schools, countries, roles (Alumni vs Student). Alumni and Students can respond to polls or acknowledge announcements.

An admin can track the progress on created polls, and delete them.

![image](https://user-images.githubusercontent.com/31665569/104497643-aa012300-55a8-11eb-88d4-5af0feac31ae.png)
Admin POV (the admin view gets little love, so the poll form / table looking sub-par is ok)

![image](https://user-images.githubusercontent.com/31665569/104497746-d0bf5980-55a8-11eb-88fd-51be99270f7b.png)
Alumnus / Student POV

A poll carousel should appear if alumni/students have polls or announcements queued for them. Answering a poll or acknowledge an announcement should bring the next slide into view, until there are no more polls

Test steps:
1. As admin, create a poll and target by:
- school
- country
- role
2. Log in as the targeted users and confirm that the targeted user can see and respond to the polls
3. Confirm that the admin can track poll responses
4. Confirm that the admin can delete a poll
